adapters/coap/src/test/java/org/eclipse/hono/adapter/coap/CoapContextTest.java

```diff
@@ -1,13 +1,13 @@
 /**
  * Copyright (c) 2020, 2023 Contributors to the Eclipse Foundation
- *
+ * <p>
```

Comment:
please remove these tags for consistency with the other file headers

Reply:
done